pkg/controller/volume/persistentvolume/volume_host.go

```diff
@@ -38,6 +39,10 @@ import (
 
 var _ vol.VolumeHost = &PersistentVolumeController{}
 
+func (ctrl *PersistentVolumeController) GetTracerProvider() trace.TracerProvider {
+	return nil
```

Comment:
Why don't we have a TracerProvider here?

Reply:
The `persistentvolume ` package is used by `kube-controller-manager`, and since this pr is focusing on add trace for csi-plugin in `kubelet`, so I do not implement the `GetTracerProvider` interface used by `kube-controller-manager.`

Reply:
got it, thanks.  Please use a Noop instead of nil

Reply:
Done